src/main/scala/edu/ie3/simona/config/ArgsParser.scala

```diff
@@ -30,7 +30,7 @@ object ArgsParser extends LazyLogging {
       nodePort: Option[String] = None,
       seedAddress: Option[String] = None,
       useLocalWorker: Option[Boolean] = None,
-      tArgs: Map[String, String] = Map.empty,
+      tArgs: Map[String, String] = Map.empty, // TODO: can be removed?
```

Comment:
Think so, but separate issue and then remove this todo?

Reply:
See #1178